Sources/BadgeGenerator/BadgeGenerator.swift

```diff
@@ -110,3 +110,96 @@ extension UIView {
     }
     
 }
+
+#if DEBUG
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+struct BadgeView: UIViewRepresentable {
+    
+    private let view = UIView()
+    
+    init(view: (UIView) -> ()) {
+        view(self.view)
+    }
+    
+    func makeUIView(context: Context) -> UIView {
+        view.layer.cornerRadius = 15
+        return view
+    }
+    
+    func updateUIView(_ view: UIView, context: Context) {}
+}
+
+fileprivate extension UIColor {
+    static let majorelle = UIColor(red: 0.36, green: 0.40, blue: 0.91, alpha: 1.0)
+    static let flamenco = UIColor(red: 0.92, green: 0.51, blue: 0.30, alpha: 1.0)
+    static let wisteria = UIColor(red: 0.81, green: 0.64, blue: 0.83, alpha: 1.0)
+    static let clamShall = UIColor(red: 0.84, green: 0.70, blue: 0.63, alpha: 1.0)
+    static let lilac = UIColor(red: 1.00, green: 0.66, blue: 0.67, alpha: 1.0)
+}
+
+fileprivate extension UIView {
```

Comment:
Good job using `fileprivate`, you could also consider using the factory pattern by using static properties on the `BadgeView` and referencing them with `Self`

Reply:
Thanks, that's a nice implementation to add to my toolset.